packages/api/tests/endpoints/test-granules.js

```diff
@@ -405,13 +405,13 @@ test.serial('CUMULUS-912 DELETE with pathParameters.granuleName set and with an
 
 test.serial('GET returns an existing granule', async (t) => {
   const response = await request(app)
-    .get(`/granules/${t.context.fakeGranules[0].granuleId}`)
+    .get(`/granules/${t.context.fakePGGranules[0].granule_id}`)
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
   const { granuleId } = response.body;
-  t.is(granuleId, t.context.fakeGranules[0].granuleId);
+  t.is(granuleId, t.context.fakePGGranules[0].granule_id);
```

Comment:
I'm thinking we should consider updating this to evaluate the entire object coming back from the API, not just the granule_id`.     It does double up a bit on the logic eval for the db translate units and/or some integrations might fail if this is wrong, but explicit > assumed coverage feels right (for the moment). 

In other words 'GET returns an existing granule' to 'GET returns the expected exisitng granule'

Reply:
Yup! Makes sense.

https://github.com/nasa/cumulus/pull/2415/commits/905a72ed8342dbfc8dc297165c6905d1e7a12703

I also updated a couple granuleIds there just to make them more realistic.